README.md

```diff
@@ -1,23 +1,23 @@
 <p align="center">
-  <a target="_blank" href="https://transform.co/metricflow">
+  <a target="_blank" href="https://www.getdbt.com/">
     <picture>
-      <img  alt="metricflow logo" src="https://github.com/transform-data/metricflow/raw/main/assets/MetricFlow_logo.png" width="auto" height="120">
+      <img  alt="metricflow logo" src="https://github.com/dbt-labs/metricflow/raw/main/assets/MetricFlow_logo.png" width="auto" height="120">
     </picture>
   </a>
   <br /><br />
   <b>Build and maintain all of your metric logic in code.</b>
   <br /><br />
-  <a target="_blank" href="https://twitter.com/transformio">
-    <img src="https://img.shields.io/twitter/follow/transformio?labelColor=image.png&color=163B36&logo=twitter&style=flat">
+  <a target="_blank" href="https://twitter.com/dbt_labs">
+    <img src="https://img.shields.io/twitter/follow/dbt_labs?labelColor=image.png&color=163B36&logo=twitter&style=flat">
   </a>
-  <a target="_blank" href="http://community.transform.co/metricflow-signup">
+  <a target="_blank" href="https://www.getdbt.com/community/">
```

Comment:
Hm, ideally we'd be able to link to the metricflow-under-dbt community Slack whenever that materializes.

Reply:
We have channels within the dbt slack community that will be dedicated to MetricFlow but that doesn't exist quite yet. I will make sure to update this link in the future when it does